java/system-test/system-test-suite/src/main/java/sleeper/systemtest/suite/dsl/SystemTestReporting.java

```diff
@@ -16,17 +16,35 @@
 
 package sleeper.systemtest.suite.dsl;
 
+import sleeper.systemtest.drivers.ingest.IngestReportsDriver;
 import sleeper.systemtest.drivers.instance.ReportingContext;
+import sleeper.systemtest.drivers.instance.SleeperInstanceContext;
+import sleeper.systemtest.drivers.partitioning.PartitionReportDriver;
+import sleeper.systemtest.drivers.util.TestContext;
 
 public class SystemTestReporting {
 
+    private final SleeperInstanceContext instance;
+    private final SystemTestClients clients;
     private final ReportingContext context;
 
-    public SystemTestReporting(ReportingContext context) {
+    public SystemTestReporting(SleeperInstanceContext instance, SystemTestClients clients, ReportingContext context) {
+        this.instance = instance;
+        this.clients = clients;
         this.context = context;
     }
 
     public void startRecording() {
         context.startRecording();
     }
+
+    public void printIngestTasksAndJobs(TestContext testContext) {
+        context.print(testContext,
+                new IngestReportsDriver(clients.getDynamoDB(), clients.getSqs(), clients.getEmr(), instance)
```

Comment:
Should this also include EMR Serverless?

Reply:
We haven't implemented reporting for pending EMR Serverless jobs yet. I think the reason EMR is included in the report is because jobs can queue up in the persistent cluster and not get done. Is that a concern for EMR Serverless?